lib/install/management/appliance.go

```diff
@@ -1018,15 +1023,30 @@ func (d *Dispatcher) CheckServiceReady(ctx context.Context, conf *config.Virtual
 	// vic-init will try to reach out to the vSphere target.
 	log.Info("Checking VCH connectivity with vSphere target")
 	// Checking access to vSphere API
-	if cd, err := d.CheckAccessToVCAPI(d.ctx, d.appliance, conf.Target); err == nil {
-		code := int(cd)
-		if code > 0 {
-			log.Warningf("vSphere API Test: %s %s", conf.Target, diag.UserReadableVCAPITestDescription(code))
-		} else {
-			log.Infof("vSphere API Test: %s %s", conf.Target, diag.UserReadableVCAPITestDescription(code))
-		}
-	} else {
-		log.Warningf("Could not run VCH vSphere API target check due to %v", err)
+
+	err := retry.Do(
```

Comment:
This is also a bummer to me too because frankly I think these little anonymous functions embedded in the call to `retry.Do` are cool :(